browser/main/TopBar/index.js

```diff
@@ -34,9 +37,52 @@ class TopBar extends React.Component {
     ee.off('top:focus-search', this.focusSearchHandler)
   }
 
+  handleKeyDown (e) {
+    // reset states
+    this.setState({
+      isAlphabet: false,
+      isIME: false
+    })
+
+    // When the key is an alphabet, del, enter or ctr
+    if (e.keyCode <= 90) {
```

Comment:
This does not work at Umlaut... Need to change.

Reply:
I decided to extend the keyCode scope to 186<=e.keyCode<=222 according to http://www.javascripter.net/faq/keycodes.htm.